v3/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/Duet3D/DSF-APIs/godsfapi/v3
+module github.com/Duet3D/
```

Comment:
This is missing the exact path to the module and will prevent compiling.

Reply:
fixed.